substrate/pkg/controller/substrate/cluster/config.go

```diff
@@ -207,9 +207,14 @@ func (c *Config) ensureBucket(ctx context.Context, substrate *v1alpha1.Substrate
 	} else {
 		logging.FromContext(ctx).Infof("Created s3 bucket %s", aws.StringValue(discovery.Name(substrate)))
 	}
+	//sets tags on a bucket. Any existing tags are replaced.
+	bucket := discovery.Name(substrate)
+	svc := s3.New(session.New())
+	putInput := &s3.PutBucketTaggingInput{Bucket: bucket, Tagging: &s3.Tagging{TagSet: []*s3.Tag{{
+		Key: aws.String("kit.aws/substrate"), Value: aws.String("Kitctl")}}}}
+	svc.PutBucketTagging(putInput)
```

Comment:
Catch error here like- 
```
if err := svc.PutBucketTagging(putInput); err != nil { 
      return fmt.Errorf("adding tag %w", err)
}
```

Reply:
Also consider passing the `&s3.PutBucketTaggingInput` directly inline to the `PutBucketTagging` call